client/webserver/site/src/js/app.js

```diff
@@ -658,6 +658,18 @@ export default class Application {
     return o
   }
 
+  haveAssetOrders (assetID) {
```

Comment:
Snuck in latest commit in https://github.com/decred/dcrdex/pull/1439?

Reply:
Oops